MIGRATION.md

```diff
@@ -10,3 +10,54 @@ Before you begin the migration process, please ensure that you have done the nec
 For existing resources created with the old provider, you'll need to import them into your new configuration. Terraform provides a feature for importing existing resources and auto-generating new Terraform configuration files. To generate configuration code for the imported resources, refer to the official [Terraform documentation](https://developer.hashicorp.com/terraform/language/import/generating-configuration) for step-by-step guidance.
 
 Once the configuration is generated, compare the generated file with your existing configuration. Be aware that field names may have changed so you should adapt the configuration accordingly. However, not all attributes from the generated configuration are needed for managing the infrastructure, meaning this set of fields can be reduced to the relevant ones from your previous configuration. Check the Terraform plan for the imported resource to identify any differences.
+
+### Example (DNS service)
```

Comment:
Can you also add an example for the Kubernetes service?

The process is the same but I think it would be more helpful since users would definitely need to migrate their cluster.

(Note: DNS is not part of the community provider)

Reply:
Ah right, I'll replace it with a SKE and LogMe examples then

Reply:
And there we could also highlighting the current issue with the import of DSAs (which will be fixed next sprint)